src/utils/libftdi_serial.cpp

```diff
@@ -72,17 +58,34 @@ void LibFTDISerial::log_error (const char *action, const char *message)
 
 int LibFTDISerial::open_serial_port ()
 {
+    if (!lib_init)
+    {
+        // failed to init libftdi; at least check the port name
+        if (description.size () == 0 || description[0] == '/')
```

Comment:
Note to self: the indexing check should be protected here; move it to the next if block